sources/packages/web/src/types/contracts/student/ApplicationContract.ts

```diff
@@ -209,3 +216,46 @@ export interface PartTimeAssessment extends BaseAssessment {
  * assessment payload created by camunda workflow.
  */
 export type Assessment = FullTimeAssessment | PartTimeAssessment;
+
+export interface ApplicationDetailHeader {
+  applicationNumber: string;
+  applicationInstitutionName: string;
+  applicationOfferingIntensity: OfferingIntensity;
+  applicationStartDate: Date | string;
+  applicationEndDate: Date | string;
+}
+
+export interface SuccessWaitingStatus {
+  success: boolean;
+  waiting: boolean;
+}
+
+export interface InProgressApplicationDetailsAPIOutDTO {
+  id: number;
+  applicationStatus: ApplicationStatus;
+  pirStatus: ProgramInfoStatus;
+  PIRDeniedReason: string;
+  offeringStatus: OfferingStatus;
+  exceptionStatus: ApplicationExceptionStatus;
```

Comment:
Please review the possible nullable properties in this object. probably all but `id`, `applicationStatus` and `pirStatus` must be nullable.